components/Snackbar/src/MDCSnackbarMessageView.m

```diff
@@ -46,7 +46,7 @@
 /**
  Shadow coloring.
  */
-static const CGFloat kShadowAlpha = 0.24f;
+static const CGFloat kShadowAlpha = (CGFloat)0.24;
```

Comment:
make this a float instead as it is used only in a float context

Reply:
If this is important, it can be done in a follow-up PR. I don't want to make refactorings if they aren't really necessary. I don't mind cleaning-up the wonky results of the automated change, but this is actually changing the type in the code.